scripts/provisioning/_s3_setup

```diff
@@ -49,6 +49,9 @@ def main():
   testCmd = subparsers.add_parser('reset', help='setup reset')
   testCmd.add_argument("--config", help='config URL', type=str, required=True)
 
+  testCmd = subparsers.add_parser('prepare', help='setup prepare')
+  testCmd.add_argument("--config", help='config URL', type=str, required=True)
```

Comment:
I think lets make '--config' as optional, not mandatory, as we are really not using the option in prepare.

Reply:
Lets keep this as it is as of now. as this is just prototype. 
Once we start implementing prepare phase, we can take the decision to make it optional or not.